framework/mongo/utils.py

```diff
@@ -20,15 +20,40 @@ def to_mongo(item):
         item = item.replace(key, value)
     return item
 
+
 def to_mongo_key(item):
     return to_mongo(item).strip().lower()
 
+
 def from_mongo(item):
     for key, value in mongo_map.items():
         item = item.replace(value, key)
     return item
 
+
 sanitize_pattern = re.compile(r'<\/?[^>]+>')
 def sanitized(value):
     if value != sanitize_pattern.sub('', value):
         raise ValidationValueError('Unsanitary string')
+
+
+def unique_on(*groups):
```

Comment:
In case we ever need to add other keyword arguments to this decorator, perhaps we should pass `groups` as a list rather than splatting it?